vpr/src/place/initial_placement.cpp

```diff
@@ -147,6 +147,26 @@ static bool is_loc_legal(const t_pl_loc& loc,
                          const PartitionRegion& pr,
                          t_logical_block_type_ptr block_type);
 
+/**
+ * @brief  Helper function to choose a subtile in specified location if compatible and available one exits.
+ *  
+ *   @param centroid The centroid location at which the subtile will be selected using its x,y, and layer.
+ *   @param block_type Logical block type of the macro head member.  
+ *   @param block_loc_registry Placement block location information. To be filled with the location
+ *   where pl_macro is placed.
+ *   @param pr The PartitionRegion of the macro head member - represents its floorplanning constraints, is the size of
+ *   the whole chip if the macro is not constrained.
+ *   @param rng A random number generator to select subtile from available and compatible ones.
+ *
+ * @return False if location on chip, legal, but no available subtile found. True otherwise. False leads us to
```

Comment:
if the location is on the chip and legal but no available subtile is found at that location. 

I'd delete the "False leads us to neighbour placement currently"